src/index.ts

```diff
@@ -2,3 +2,13 @@ import './polyfills'
 import './components/shop'
 import './components/checkout-button'
 import './shop'
+import { Shop } from './shop'
+
+// Shop.doSomething(Math.random())
+
+export { Shop } from './shop'
+console.log('################')
+export default Shop
+
+// only this one works so far! :(
+window['Shop'] = Shop
```

Comment:
Still needed ^ ?

Reply:
unfortunately yes, TODO added, needs fixing, if you have time, please have a look. I tried for quite a while but couldn't get it to work. IMO, it has to do with webpack, not TS. How to import a class from the web-pack built?